lib/suppressions.cpp

```diff
@@ -173,6 +173,16 @@ std::string Suppressions::addSuppressionLine(const std::string &line)
     Suppressions::Suppression suppression;
     if (std::getline(lineStream, suppression.errorId, ':')) {
         if (std::getline(lineStream, suppression.fileName)) {
+            size_t hashpos, cppcommentpos, endpos;
```

Comment:
I'd rather avoid a declaration with delayed initialization

Reply:
Fixed.  Also changed to std::string::size_type as used elsewhere  in the code.